.changes/next-release/service.internal.s3shared-feature-1612556486110454000.json

```diff
@@ -0,0 +1,9 @@
+{
+ "ID": "service.internal.s3shared-feature-1612556486110454000",
+ "SchemaVersion": 1,
+ "Module": "service/internal/s3shared",
+ "Type": "feature",
+ "Description": "adds support for s3 vpc endpoint interface",
```

Comment:
Nit this probably should call out that the S3 clients will no longer validate the ARN partition and client's configured region partition match. Instead allowing the API call to be made, and potentially fail if the credentials aren't valid

Reply:
My understanding of the change was incorrectly not validating partition match was only disabled when no partition is available from the base endpoint resolved